### ISSUE TITLE
[a11y]: Screen reader does not announce when both KiB in and KiB out checkboxes are selected

### ISSUE THREAD

**Issue by anamhussain94** [carbon-design-system/carbon#20476]
### Package

carbon-components

### Browser

Chrome

### Operating System

Windows, MacOS

### Package version

v1.23.17

### Automated testing tool and ruleset

WCAG Mapping: 4.1.2 Name, Role, Value

### Description

The issue is with https://www.npmjs.com/package/@carbon/charts-react 

<img width="1920" height="961" alt="Image" src="https://github.com/user-attachments/assets/d4330630-729f-4403-a405-e71863434fa0" />

This overall functionality of the checkboxes is still confusing the users who rely on the screen readers . The visual indication of the checkboxes are not clearly communicated with by the screen reader
Hence its a valid issue against WCAG guidelines.

### WCAG 2.1 Violation

WCAG Mapping: 4.1.2 Name, Role, ValueWCAG Mapping: 4.1.2 Name, Role, Value

### Reproduction/example

https://www.npmjs.com/package/@carbon/charts-react

### Steps to reproduce

https://dev.console.test.cloud.ibm.com/infrastructure
VPNs for VPC - Client to site servers - Details Page - Monitoring

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
Hey, I believe this issue is more related to [Carbon Charts](https://github.com/carbon-design-system/carbon-charts). I'll forward it to the team, they’ll be better able to help you out.
